third_party/BUILD.four_seasons_snippet

```diff
@@ -0,0 +1,6 @@
+
+filegroup(
```

Comment:
It doesn't seem like this file is needed because you're using `//third_party:BUILD.zip_file`